loader/terminator.c

```diff
@@ -128,15 +128,17 @@ VKAPI_ATTR void VKAPI_CALL terminator_GetPhysicalDeviceFeatures2(VkPhysicalDevic
     struct loader_icd_term *icd_term = phys_dev_term->this_icd_term;
     const struct loader_instance *inst = icd_term->this_instance;
 
+    assert(inst != NULL);
+
     // Get the function pointer to use to call into the ICD. This could be the core or KHR version
     PFN_vkGetPhysicalDeviceFeatures2 fpGetPhysicalDeviceFeatures2 = NULL;
-    if (inst != NULL && inst->enabled_known_extensions.khr_get_physical_device_properties2) {
-        fpGetPhysicalDeviceFeatures2 = icd_term->dispatch.GetPhysicalDeviceFeatures2KHR;
-    } else {
+    if (inst->app_api_major_version > 1 || inst->app_api_major_version >= 1) {
```

Comment:
This should be `app_api_minor_version` in the second condition, right? (and this applies to every other terminator in this file and elsewhere I think...)

Reply:
🤯